tests/test_symbolics.py

```diff
@@ -15,6 +15,7 @@
                               INT, FieldFromComposite, IntDiv, Namespace, Rvalue,
                               ReservedWord, ListInitializer, uxreplace,
                               retrieve_derivatives, BaseCast)
+from devito.symbolics.unevaluation import Mul
```

Comment:
`from devito.symbolics.unevaluation import Mul as UnevalMul` would prevent potential gotchas down the line